draft-ietf-quic-transport.md

```diff
@@ -444,7 +444,10 @@ Google QUIC Demultipexing Bit:
 Reserved:
 
 : The sixth, seventh, and eighth bits (0x7) of octet 0 are reserved for
-  experimentation.
+  experimentation.  Endpoints can MUST ignore these bits unless they are
```

Comment:
c̶a̶n̶

gQUIC 44 is using the Long Header according to draft-11 (?), where the last two bits are used to encode the packet number length. So technically, the bits are not ignored there.

Reply:
This is version-specific text, so I think that we're OK.  Even if qQUIC doesn't use the invariants (yet) to identify versions.